### PR TITLE
update gimvi and LinearSCVI tutorial

### PR DESCRIPTION
**Pull request by galenxing**

### CODE REVIEW

File: gimvi_tutorial.ipynb [hunk 1]
```diff
@@ -9,7 +9,7 @@
    },
```
Comment (adamgayoso) [inline gimvi_tutorial.ipynb:9]: <p>should the increment batch_idx be happening inside gimvi?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (adamgayoso) [inline gimvi_tutorial.ipynb:9]: <p>I think you have to specify <code>use_rep="X"</code>	in the call to neighbors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (galenxing) [reply gimvi_tutorial.ipynb]: hidden

File: linear_decoder.ipynb [hunk 2]
```diff
@@ -1,702 +1,900 @@
 {
```
Comment (adamgayoso) [inline linear_decoder.ipynb]: <p>let's just import scvi for simplicity, </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (adamgayoso) [inline linear_decoder.ipynb]: <p>let's use the same preprocessing here as we do in the quick start, (subset option) no need for this dense step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (adamgayoso) [inline linear_decoder.ipynb]: <p>add label for train and test set and legend (quick start has this)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (adamgayoso) [inline linear_decoder.ipynb]: <p>I think loadings is a pd dataframe already so the latter part is not necessary.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (adamgayoso) [inline linear_decoder.ipynb]: <p>min dist = 0.3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Review by adamgayoso — APPROVED** [review]

**Pull request merged**